server/utils/tasks/listTable.test.ts

```diff
@@ -197,16 +198,52 @@ describe('table', () => {
   })
 
   describe('nameAnchorCell', () => {
-    it('returns the cell when there is a person present in the task', () => {
+    it('returns the name when the person summary is FullPersonSummary  in the task', () => {
+      const personSummary = fullPersonSummaryFactory.build()
       const task = taskFactory.build({
         taskType: 'Assessment',
+        personSummary,
       })
       expect(nameAnchorCell(task)).toEqual({
         html: linkTo(
           paths.tasks.show,
           { id: task.id, taskType: kebabCase(task.taskType) },
           {
-            text: task.personName,
+            text: personSummary.name,
+            attributes: { 'data-cy-taskId': task.id, 'data-cy-applicationId': task.applicationId },
+          },
+        ),
+      })
+    })
+    it('returns the lrn crn when the person summary is RestrictedPersonSummary  in the task', () => {
```

Comment:
Please change this to `returns the Limited Access Offender (LAO) CRN when the person summary is RestrictedPersonSummary in the task`